.scripts/release_NupkgAudit.ps1

```diff
@@ -165,6 +165,15 @@ function Get-DoesXmlDocContainsLang ([string]$dllPath) {
     Test-Condition ($result) $message $requirement;
 }
 
+function Get-DoesPdbExist ([string]$dllPath) {
+    # CONFIRM .PDB SYMBOLS EXIST WITH EACH DLL
+    [string]$docFile = $dllPath -replace ".dll", ".pdb";
```

Comment:
If `$dllPath` contains `.dll` in the file name or folder name, they will be replaced, too.

Reply:
This is currently not an issue. 
In this scenario, i've just unpacked a NUPKG and am inspecting that each sub directory contains one of each (dll, xml, pdb).
Since i'm unzipping the NUPKG, I have high control of the file path and NUPKGs themselves don't contain ".dll" in their paths.

I think it's safe to ignore this for now. 
If someone makes a change to our build system or paths it would be a breaking change at that time.